nipype/interfaces/utility/base.py

```diff
@@ -1,16 +1,21 @@
 # -*- coding: utf-8 -*-
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
-"""Various utilities
+"""
+Various utilities
 
    Change directory to provide relative paths for doctests
-   >>> import os
-   >>> filepath = os.path.dirname( os.path.realpath( __file__ ) )
-   >>> datadir = os.path.realpath(os.path.join(filepath, '../testing/data'))
-   >>> os.chdir(datadir)
+
+   .. testsetup::
```

Comment:
I thought we were printing those code-blocks in the documentation, but now I see we must be manually parsing them out, right?.

The ```.. testsetup::``` directive just makes sure that the code is run before doctests but not printed out in the documentation.

I guess this would be more appropriate, since sphinx has a directive for this use case of docstrings. But I don't mind to roll it back if you want :).